proto/rill/runtime/v1/api.proto

```diff
@@ -84,6 +84,16 @@ service RuntimeService {
     };
   }
 
+  // ListExamples lists all the examples embedded into binary
+  rpc ListExamples(ListExamplesRequest) returns (ListExamplesResponse) {
+    option (google.api.http) = {get: "/v1/examples"};
+  }
+  
+  // UnpackExample unpacks an example project
+  rpc UnpackExample(UnpackExampleRequest) returns (UnpackExampleResponse) {
+    option (google.api.http) = {get: "/v1/instances/{instance_id}/files/unpack-example"};
```

Comment:
Should be `post` instead of `get` (note: see other examples, it then also needs a `body` param)